spec/cc/engine/csslint_spec.rb

```diff
@@ -16,6 +16,11 @@ module Engine
           expect{ lint.run }.to output(/Don't use IDs in selectors./).to_stdout
         end
 
+        it 'fails on malformed file' do
+          create_source_file('foo.css', '')
```

Comment:
Github is showing an empty string but there are some binaries in there. I copied it from the malformed file that caused the issue on my codebase. I'll check whether we can achieve the same error by clearer means.